bench-tps/src/bench.rs

```diff
@@ -607,11 +607,10 @@ impl<'a> FundingTransactions<'a> for Vec<(&'a Keypair, Transaction)> {
 
     fn send<T: BenchTpsClient>(&self, client: &Arc<T>) {
         let mut send_txs = Measure::start("send_txs");
```

Comment:
Nit: is this supposed to measure only the send time or the send time + the time clone and collect the transactions?

Reply:
both I guess. re-name it `send_and_clone_txs` ?

Reply:
Sounds good, please rename. Other than that, LGTM.